charts/pythie/templates/pythie-serving-deployment.yaml

```diff
@@ -102,7 +102,11 @@ spec:
               value: {{ $servingValue.grpcServerTimeoutSeconds }}
             {{- end }}
             - name: "VERBOSITY_LEVEL"
+            {{- if $.Values.pythieServing.verbosityLevel }}
```

Comment:
instead set `pythieServing.verbosityLevel: 0` in values.yaml and then here just put:
```
- name: "VERBOSITY_LEVEL"
  value: {{ $.Values.pythieServing.verbosityLevel }}
```

Reply:
its okay like this!